diff --git a/build_files/shared/build-dx.sh b/build_files/shared/build-dx.sh
index e7d9510..f1d030f 100755
--- a/build_files/shared/build-dx.sh
+++ b/build_files/shared/build-dx.sh
@@ -31,6 +31,9 @@ EOF
 # Install AKMODS
 /ctx/build_files/dx/02-install-kernel-akmods-dx.sh
 
+# Install Chrome
+/ctx/build_files/dx/02-enable-chrome-yum-repo.sh
+
 # Install Packages
 /ctx/build_files/dx/03-packages-dx.sh
 
